chapters/annotations.tex

```diff
@@ -1544,11 +1544,11 @@ \subsubsection{Conversion rules}\doublelabel{conversion-rules}
 class-parameters, or hierarchical names. For hierarchical names the
 longest match is used first.
 
-For replaceable classes in packages (and other class-parameters) \lstinline!convertElement! shall
-be used if the class is renamed within the package, whereas \lstinline!convertClass! shall only be used if the class
-is placed outside of the package.
+For replaceable classes in packages (and replaceable classes in other classes) \lstinline!convertElement! shall
+be used if the class is renamed within the package (or class), whereas \lstinline!convertClass! shall only be used if the class
+is placed outside of the package (or class).
```

Comment:
If I understood correctly, in the phone meeting we said that "placed outside of the package" means in the new library version the replaceable class will not be in the same package (or class) anymore. Is it right? Should we add it explicitly?

Reply:
That is correct, I don't know how to make that much clearer than "placed outside of the package (or class)".

Reply:
How about:
whereas \lstinline!convertClass! shall only be used if in the new library version the class is moved outside of its package (or class).

Reply:
Hmm... I see the basic idea, but I find that specific text misleading since the "package" in "its package" is the package for the old library version, and "its package" for the new version would normally to package the new version is in.

Reply:
True.